understanding/on-focus.html

```diff
@@ -83,7 +83,7 @@ <h2>事例</h2>
                <li><p><strong>失敗例: ヘルプのダイアログ</strong></p>
                   
                   
-                  <p>フィールドがフォーカスを受け取ると、フィールドを説明し、オプションを提供するヘルプのダイアログウィンドウが開く。キーボードの利用者がウェブページの中をタブ移動すると、ダイアログが開いて、利用者がフィールドを過ぎてタブを移動しようとするたびにコントロールからキーボード フォーカスが離れる。</p>
+                  <p>フィールドがフォーカスを受け取ると、そのフィールドを説明し、オプションを提供するヘルプのダイアログウィンドウが開く。キーボードの利用者がウェブページ内をタブで移動すると、ダイアログが開いて、利用者がそのフィールドの先にタブで移動しようとするたびに、キーボードのフォーカスがコントロールから離れる。</p>
```

Comment:
この事例は失敗例なのですが、「フィールドの先にタブで移動しようとするたびに、キーボードのフォーカスがコントロールから離れる」というのは何が問題なのかよくわからない感じがします。

>When a field receives focus, a help dialog window describing the field and providing options opens. As a keyboard user tabs through the Web page, the dialog opens, moving the keyboard focus away from the control every time the user attempts to tab past the field.

past は「通過」と訳すと良いのかもしれません。あと every time をもう少し強めに訳す?
たとえばこんな感じでしょうか。

```suggestion
                  <p>フィールドがフォーカスを受け取ると、そのフィールドを説明し、オプションを提供するヘルプのダイアログウィンドウが開く。キーボードの利用者がウェブページ内をタブで移動すると、そのダイアログが開いて、利用者がそのフィールドをタブ移動で通過しようとするたびに毎回、キーボードのフォーカスがコントロールから離れる。</p>
```